### PR TITLE
Utilities to work with Durations as numbers

### PR DESCRIPTION
**Pull request by jodastephen**

### CODE REVIEW

**Comment by jodastephen** [comment]
I think the latest commit responds to the review here and on the associated issue. Let me know!

**Comment by cowwoc** [comment]
@perceptron8 Thoughts?

**Comment by perceptron8** [comment]
@cowwoc LGTM, I'm more than happy with @jodastephen's changes. Sorry for not responding earlier!

**Comment by cowwoc** [comment]
@perceptron8 Perfect, thanks :)

@jodastephen Sounds like you're good to go.

File: src/main/java/org/threeten/extra/Temporals.java [hunk 1]
```diff
@@ -455,13 +455,15 @@ public static Duration durationFromDoubleSeconds(double seconds) {
...
+     * See {@link #durationToBigDecimalSeconds(Duration)} and {@link #durationFromBigDecimalSeconds(BigDecimal)}.
+     * Note that due to the rounding up, 1 nanosecond multiplied by any number smaller than 1 will still be 1 nanosecond.
      * 
      * @param duration  the duration to multiply, not null
      * @param multiplicand  the multiplication factor
      * @return the multiplied duration, not null
      */
     public static Duration multiply(Duration duration, double multiplicand) {
-        if (multiplicand == 0d) {
+        if (multiplicand == 0d || multiplicand == -0d || duration.isZero()) {
```
Comment (cowwoc) [inline src/main/java/org/threeten/extra/Temporals.java:466]: Intellij claims `multiplicand == -0d` is always false.

**Pull request merged**